assets/js/atomic/components/product/button/index.js

```diff
@@ -166,7 +167,9 @@ const ProductButton = ( { product, className } ) => {
 			{ allowAddToCart ? (
 				<button
 					onClick={ addToCart }
-					aria-label={ productCartDetails.description }
+					aria-label={ decodeEntities(
+						productCartDetails.description
+					) }
```

Comment:
I don't think it will be an issue, but we should probably test to make sure html attributes aren't broken if the encoded entities are quotes.  I think we'll be fine because react sanitizes that.

Reply:
Confirmed React is re-encoding quote characters before rendering.